docker/dockerfiles/Dockerfile.server

```diff
@@ -47,5 +48,7 @@ COPY views views
 
 RUN pip install -e .
 
-ENV CODALAB_HOME=/home/codalab
```

Comment:
Why did we remove this line?

Reply:
Want to have one place where things are set rather than having defaults in various places spread out.